### PR TITLE
Create output.js

### PR DESCRIPTION
**Pull request by ntwallace**

### CODE REVIEW

**Comment by ntwallace** [comment]
Output file we sketched out while working on the diagram, it's a mix of actual code and commented out psuedo code at the moment.

File: public/output.js [hunk 1]
```diff
@@ -0,0 +1,140 @@
+// Open and connect output socket
+let socket = io('/output');
+
+// Listen for confirmation of connection
+socket.on('connect', function() {
+  console.log("Connected");
+});
+
+let users = {}; // track users
+let flipOrder[]; // track order users flipped their phones in
```
Comment (79) [inline public/output.js]: Whoa does this (`[]`) work in javascript?

File: public/output.js [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+}
+
+// create new user
+function createNewUser(id, user) {
+  users[id] = {
+    username: user,
+    myTurn: false,
+    shook: true,
+    flipped: false,
+    burgerLives = 3
```
Comment (79) [inline public/output.js]: /s/=/:

I always do this, too. =P

File: public/output.js [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+  // listen for socket.on('flip', ...) events from all users
+  // as each user meets the flip event criteria, "unshift" their socket.id into the flipOrder array
+  // as a result, whoever the last user was to flip their phone will be the last socket.id in the array
+  // after x amount of seconds (3? 5?), get the last item in the flipOrder array (flipOrder[-1])
+  // call loseLife(flipOrder[-1]);
+  // clear flipOrder;
+  //
+  // return to generateImage()
+
+  /* unshift() works the opposite of push() -- it adds values to the end of an array
```
Comment (79) [inline public/output.js]: I think this is backwards. `push` adds to the end of an array and `unshift` adds the beginning.

File: public/output.js [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+  array.unshift(0)
+  array = [1, 2, 3, 4, 0] */
+
+}
+
+function generateImage() { // this function generates a random image (burger, fries, etc) on screen
+   // generate random number 1-6
+   // burger = 1, other images = 2-6
+  let imageChoice; 
+  imageChoice = (Math.floor(Math.random() * 6)) + 1;
```
Comment (79) [inline public/output.js]: p5 has a nice `random` method to clean this up, i.e. `random(1, 6)`

**Review by 79 — APPROVED** [review]
Great start! I added some comments because I'm a neckbeard and thought it would be good for the team to see github pull-requests in full action.

I'll leave this up so we can merge Sunday.

File: public/output.js (hunk 1 above)
Comment (ntwallace) [reply public/output.js]: Lol no, should be let flipOrder = [];

File: public/output.js (hunk 3 above)
Comment (ntwallace) [reply public/output.js]: Yep I've got it backwards

File: public/output.js (hunk 4 above)
Comment (ntwallace) [reply public/output.js]: Oh nice, ok. I know next to nothing about p5.

**Pull request merged**